### ISSUE TITLE
Unexpected token '\0' 

### ISSUE THREAD

**Issue by alundiak** [CSSLint/csslint#604]
Indeed this is CSS hack, but I need in my code:

```css
.dropdown .text-field .ghost {
  margin-top: -2px;
  margin-top: -3px\0; /*For IE*/
}
```

And csslint tells me:

```
Unexpected token '3px\0' at line ...
  margin-top: -3px\0;
```
Similar answer tells me `grunt-contrib-csslint`.

Is this a designed behavior?

**Comment by aniude** [comment]
I got the something problem. anyone can help ?

**Comment by mlh758** [comment]
CSSLint also does this for the "\9" hack.

**Comment by cscott** [comment]
It seems perhaps CSSLint should have a `/* csslint ignore:line */` option for these sorts of hacks?

**Comment by ideadapt** [comment]
btw: the \9 issue was already reported in #337

**Comment by mlh758** [comment]
@ideadapt Good to know. It seems like this is tied to the same parser issue then?

**Comment by alundiak** [comment]
@frvge @XhmikosR @nschonni @gurdiga @shama @sindresorhus  any thoughts on described issue and comments above?

**Comment by shama** [comment]
@alundiak Why did you ping me on this?

**Comment by gurdiga** [comment]
I’m wondering if there is something similar to what JS linters have: you can for example disable a specific rule for a specific file/block/line by adding a magic comment. For example with ESlint you [can](http://eslint.org/docs/user-guide/configuring.html#configuring-rules) say:
```js
alert('foo'); // eslint-disable-line no-alert
```
Is there anything like this for csslint?

**Comment by frvge** [comment]
I just merged https://github.com/CSSLint/csslint/pull/630 to allow certain rules on a per-line basis. Documentation should be on the wiki soon.

Please use this new feature (in master) to exclude the rule on the line where it's needed. I don't think it's a good idea to allow all \[something] hacks so for now I'm closing this.

**Comment by mlh758** [comment]
Sounds like a good solution to me, thank you!
